packages/wdio-protocols/protocols/webdriver.json

```diff
@@ -175,7 +175,7 @@
       "ref": "https://w3c.github.io/webdriver/#new-window",
       "parameters": [{
         "name": "type",
-        "type": "string",
+        "type": "'tab' | 'window'",
```

Comment:
This is causing the e2e test to fail, and this is required to update the type definition. So I'm gonna close this PR then.